docs/data/material/components/text-fields/InputSuffixShrink.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { outlinedInputClasses } from '@mui/material/OutlinedInput';
+import { filledInputClasses } from '@mui/material/FilledInput';
+import { inputBaseClasses } from '@mui/material/InputBase';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+
+export default function InputSuffixShrink() {
+  return (
+    <Box
+      component="form"
+      sx={{ '& > :not(style)': { m: 1, width: '25ch' } }}
+      noValidate
+      autoComplete="off"
+    >
+      <TextField
+        id="outlined-suffix-shrink"
+        label="Outlined"
+        variant="outlined"
+        slotProps={{
+          input: {
+            endAdornment: (
+              <InputAdornment
+                position="end"
+                sx={{
+                  alignSelf: 'flex-end',
+                  opacity: 0,
+                  marginBottom: '5px',
+                  pointerEvents: 'none',
+                  [`.${outlinedInputClasses.root} &`]: {
+                    marginBottom: '16px',
+                  },
```

Comment:
Not sure if I'm missing something, but I think I get the same result removing some styles.

```suggestion
                  opacity: 0,
                  pointerEvents: 'none',
```

Reply:
That's true. Thanks.